src/index.html

```diff
@@ -1,12 +1,341 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="catalog">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header catalog__header">
+      <a class="logo" href="#Home">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+      <div class="space"></div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__item_link is-active" href="#Apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link" href="#Samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link"
+            data-qa="nav-hover"
+            href="#Laptops_&_Computers">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link" href="#Gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link" href="#Tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link" href="#Photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__item_link" href="#Video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog__content">
+      <div
+        class="card catalog__card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
```

Comment:
```suggestion
        >
        
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
        
        <p class="card__code">Product code: 195434</p>
        
        <div class="card__review">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          
          <p class="card__reviews">Reviews: 5</p>
        </div>
```
fix code style all over the page